src/bun.js/webcore/streams.zig

```diff
@@ -2891,6 +2892,11 @@ pub const FileSink = struct {
     pub const IOWriter = bun.io.StreamingWriter(@This(), onWrite, onError, onReady, onClose);
     pub const Poll = IOWriter;
 
+    export fn Bun__ForceFileSinkToBeSynchronousOnWindows(globalObject: *JSC.JSGlobalObject, jsvalue: JSC.JSValue) {
```

Comment:
`comptime std.debug.assert(Environment.isWindows)`

this should not be even looked at outside of windows